### PR TITLE
Add a script for migrating auth0 users from one auth0 tenant to another

### PR DESCRIPTION
**Pull request by evansiroky**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [ ] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

I made this script to migrate auth0 users from one tenant to another. It might be useful to keep in the repo for later.

### CODE REVIEW

File: scripts/migrate-auth0-users.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+// the below json files should have the form of:
+// {
+//   apiClient: 'SECRET',
+//   apiSecret: 'SECRET',
+//   domain: 'SECRET.auth0.com',
+//   clientId: 'SECRET'
+// }
+const from = require('./from-client.json')
+const to = require('./to-client.json')
```
Comment (landonreed) [inline scripts/migrate-auth0-users.js]: Should we add `.json.tmp` files for these? Also, it might be worth adding them to `.gitignore`.

Comment (evansiroky) [reply scripts/migrate-auth0-users.js]: I think it's enough to include the example in the comments. I sent you the secrets privately.

**Comment by evansiroky** [comment]
Just changed the travis.yml file so that the travis build runs properly.

**Pull request merged**